actionpack/lib/action_controller/metal/data_streaming.rb

```diff
@@ -79,14 +79,15 @@ def send_file(path, options = {}) #:doc:
       # <tt>render plain: data</tt>, but also allows you to specify whether
       # the browser should display the response as a file attachment (i.e. in a
       # download dialog) or as inline data. You may also set the content type,
-      # the apparent file name, and other things.
+      # the file name, and other things.
       #
       # Options:
       # * <tt>:filename</tt> - suggests a filename for the browser to use.
       # * <tt>:type</tt> - specifies an HTTP content type. Defaults to 'application/octet-stream'. You can specify
-      #   either a string or a symbol for a registered type register with <tt>Mime::Type.register</tt>, for example :json.
-      #   If omitted, type will be guessed from the file extension specified in <tt>:filename</tt>.
-      #   If no content type is registered for the extension, default type 'application/octet-stream' will be used.
+      #   either a string or a symbol for a registered type.
+      #   register a type, for example :json, with <tt>Mime::Type.register</tt>.
```

Comment:
Actually just drop `register`. The sentence goes like this-

`You can specify either a string or a symbol for a registered type with <tt>Mime::Type.register</tt>, for example :json.`